k8s/environment/environment.go

```diff
@@ -646,15 +650,32 @@ func (m *Environment) RunCustomReadyConditions(customCheck *client.ReadyCheckDat
 			return fmt.Errorf("Test must be configured in the environment when using the remote runner")
 		}
 		rrSelector := map[string]*string{pkg.NamespaceLabelKey: ptr.Ptr(m.Cfg.Namespace)}
+		// if no runner name is specified use constant
+		if m.Cfg.RunnerName == "" {
+			m.Cfg.RunnerName = REMOTE_RUNNER_NAME
+		}
 		m.AddChart(NewRunner(&Props{
-			BaseName:           REMOTE_RUNNER_NAME,
+			BaseName:           m.Cfg.RunnerName,
+			ReportPath:         m.Cfg.ReportPath,
 			TargetNamespace:    m.Cfg.Namespace,
 			Labels:             &rrSelector,
 			Image:              m.Cfg.JobImage,
 			TestName:           m.Cfg.Test.Name(),
 			NoManifestUpdate:   m.Cfg.NoManifestUpdate,
 			PreventPodEviction: m.Cfg.PreventPodEviction,
 		}))
+		if m.Cfg.ReportPath != "" {
+			m.AddChart(DataFromRunner(&Props{
```

Comment:
added this pod to enable accessing reports once remote runner job gets completed